airflow/providers/amazon/aws/sensors/sqs.py

```diff
@@ -104,14 +104,13 @@ def __init__(
 
         self.hook: Optional[SqsHook] = None
 
-    def poke(self, context: 'Context'):
+    def poll_sqs(self, sqs_conn: Any) -> Iterable:
+        """Poll SQS queue to retrieve messages
+        Args:
+            sqs_conn (Any): SQS connection
+        Returns:
+            Iterable: list of messages retrieved from SQS
```

Comment:
Not the format for docstrings that Airflow uses:

```suggestion
        """
        Poll SQS queue to retrieve messages.
        
        :param sqs_conn: SQS connection
        :return: A list of messages retrieved from SQS
```